test/functional/mempool_compatibility.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+# Copyright (c) 2017-2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test that mempool.dat is compatible between versions
+
+Download node binaries:
+contrib/devtools/previous_release.sh -b v0.19.1
+"""
+
+import os
+
+from test_framework.test_framework import BitcoinTestFramework
+
+class MempoolCompatibilityTest(BitcoinTestFramework):
+    def set_test_params(self):
+        self.num_nodes = 2
+
+    def skip_test_if_missing_module(self):
+        self.skip_if_no_wallet()
+        self.skip_if_no_previous_releases()
+
+    def setup_nodes(self):
+        self.add_nodes(self.num_nodes, versions=[
+            190100,
```

Comment:
Suggest documenting what's interesting about that version, could be a link to Github issue.